src/main/kotlin/org/objectionary/ddr/launch/Main.kt

```diff
@@ -29,6 +29,5 @@ package org.objectionary.ddr.launch
  * - args[0] path to the folder with the program to be analyzed
  */
 fun main(args: Array<String>) {
-    // launch("/home/olesya/huawei/ssh-ddr/ddr/src/test/resources/unit/in/condition_to_name")
-    launch(path = args[0])
+    DdrLaunched(path = args[0]).launch()
```

Comment:
@c71n93 Maybe not the best naming here. Why would you want to "launch()" something that's already "DdrLaunched"? Maybe we could rename `DdrLaunched` into `DdrWorkflow` or `DdrAnalysis` in which case `DdrWorkflow().lanuch()` would look natural.